docs/user/discover.asciidoc

```diff
@@ -107,11 +106,26 @@ image:images/document-table.png[Document table with fields for manufacturer, cus
 . To rearrange the table columns, click a
 column header, and then select *Move left* or *Move right*.
 
-. To view more of the document table,
+. To copy the name or values in a column to the clipboard, click the column header and select the desired **Copy** option.
```

Comment:
👍 for adding this, it might also be a fit for the "Optionally try out section" ... since the action before are essential, and I feel like the copy action is not used that often